examples/todolist/docker-compose.yml

```diff
@@ -0,0 +1,15 @@
+version: '3'
+services:
+  testcafe:
+    build:
+      context: .
+      dockerfile: testcafe.dockerfile
+    command:
+      - firefox
```

Comment:
Yes, chromium is had. We can't run two browsers in parallel because they conflict on one service